swift/WorkflowTesting/Sources/WorkflowRenderTester.swift

```diff
@@ -114,6 +149,26 @@ public final class RenderTester<WorkflowType: Workflow> {
         return self
     }
 
+    /// Convenience method for testing without creating an explicit RenderExpectation.
+    @discardableResult
+    public func render(
```

Comment:
Added this after discussion with @timdonnelly - improves the ergonomics (instead of making a full `RenderExpectations`, can just specify which expectations on the call to `render` with the tester)